ci/openstack-image-build-env

```diff
@@ -10,7 +10,7 @@ PROFILES = {
     "horizon": "apache",
     "monasca-api": "apache influxdb",
     "ironic": "ipxe ipmi qemu tftp",
-    "glance": "ceph",
+    "glance": "ceph os_brick python-cinderclient",
```

Comment:
This is `PROFILES`, you would need to add it into `PIP_PACKAGES`.

Let me work on fixing it.

Reply:
@mnaser yes my bad, sorry. I pushed a fix